webservice/src/main/java/pl/edu/pjatk/lnpayments/webservice/common/Constants.java

```diff
@@ -2,7 +2,7 @@
 
 public class Constants {
 
-    public static final String PAYMENTS_PATH = "/payments";
+    public static final String PAYMENTS_PATH = "/api/payments";
```

Comment:
Można to zrobić bardziej elegancko poprzez dodanie "server.servlet.context-path=/api" w application.properties. Inaczej musielibyśmy dodawać z ręki api do każdego, a tak ładnie w configu będzie.